.phan/config.php

```diff
@@ -43,9 +43,7 @@
     ],
 
     'suppress_issue_types' => [
-        'PhanPluginDescriptionlessCommentOnPublicMethod',
         'PhanPluginDescriptionlessCommentOnProtectedProperty',
-        'PhanPluginDescriptionlessCommentOnPublicProperty',
```

Comment:
I just noticed it's pretty easy to enforce this by fixing a few things. :) I'm happy with it.

Reply:
great